app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,35 @@
+class InvitationsController < ApplicationController
+  before_action :authenticate!
+  before_action :set_user!
+  before_action :set_invitation!
+
+  def show
+    redirect_to_organization! if @user.student_of? @organization
```

Comment:
But... why not using `http://orga.mumuki.io/join/1f3g-curso`? From my point of view it has much more sense.